routes/index.js

```diff
@@ -75,6 +75,11 @@ router.get('/overlays/springshowdown/all4by3', function(req, res) {
   res.render('overlays/springshowdown/all4by3', { title: 'Spring Showdown Overlay', layout: false });
 });
 
+/*Generic*/
+router.get('/overlays/generic/generic', function(req, res) {
+  res.render('overlays/generic/generic', { title: 'Generic Overlay', layout: false });
```

Comment:
You're right. I originally had more than one file in there, and didn't change it back.